components/network/socat/Makefile

```diff
@@ -14,20 +14,23 @@
 #
 
 PREFERRED_BITS=		64
+OPENSSL_VERSION= 3.1
```

Comment:
You can remove the PREFERRED_BITS settings as we only use 64 bit.

Reply:
Removed PREFERRED_BITS line